src/frontend/components/postsContent/PostsContent.css

```diff
@@ -7,6 +7,30 @@
   display: none;
 }
 
+.posts-tabs {
+  color: var(--text--primary--color);
+  width: 100%;
+}
+
+.tablist {
+  color: var(--text--primary--color);
+  width: 100%;
+  gap: 5rem;
+  padding: 0rem 13rem;
+  align-items: center;
+  justify-content: center;
+  border: none !important;
```

Comment:
you can remove it !important and try  others method.
you can increase specificity.